articles/custom-domains/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Custom Domains
+description: How to map custom domains
+toc: true
+---
+# Custom Domains
+
+Auth0 allows you to map your the domain for your tenant to a custom domain of your choosing. This allows you maintain a consistent experience for your users by keeping them on your domain instead of redirecting or using Auth0's domain. For example, if your Auth0 domain is `northwind.auth0.com`, you can have your users to see, use, and remain on `login.northwind.com`.
+
+## Prerequisites
+
+You'll need to register and own the domain name to which you're mapping your Auth0 domain.
+
+## Features Supporting Use of Custom Domains
+
+Currently, the following Auth0 features and flows support use of custom domains:
+
+* OAuth 2.0/OIDC-Compliant Flows (all of which make calls to the [`/authorize` endpoint](/api/authentication#authorize-client))
+* Guardian (Version 1.3.3 or later)
+* Emails -- the links included in the emails will use your custom domain
+
+## Certificates
+
+You can manage the certificates for your custom domains, or you can have Auth0 manage them on your behalf.
+
+If you opt to manage your own certificates, you'll need to use a reverse proxy that you configure using AWS CloudFront or Microsoft Azure.
+
+### Auth0-Managed Certificates
+
+::: warning
+Auth0-Managed Certificates is a beta feature.
+:::
+
+Auth0 will manage the creation and renewal of the certificates for your custom domain. This is the simplest custom domains deployment option.
+
+### Self-Managed Certificates
+
+You are responsible for managing your SSL/TLS certificates and configuring a reverse proxy to handle SSL termination and the forwarding of requests to Auth0. Choose this option if you wish to have more control of your certificates (such as choosing your own CA or certificate expiration), or if you want to have more monitoring over your API calls to Auth0.
+
+## How to Configure Custom Domains
+
+Setting up your custom domain requires you to do the following steps:
+
+1. Provide your domain name to Auth0 and verify ownership
+1. Configure the reverse proxy (if using self-managed certificates)
+1. Configure the hosted login page
+1. *Optional*: Enable custom domains in Auth0 emails
+
+### Step 1: Configure Auth0
+
+Log in to the Dashboard and go to [Tenant Settings](${manage_url}/#/tenant). Click over to the **Custom Domains** tab.
+
+![](/media/articles/custom-domains/custom-domains.png)
+
+Enter your custom domain in the provided box. Indicate whether you're using **Self-managed certificates** or **Auth0-managed certificates**. Click **Add Domain**.
+
+Before you can use this domain, you'll need to verify that you own your domain:
+
+* **Auth0-Managed Certificates**: Add the CNAME verification record listed in the Dashboard to your domain's DNS record
+
+	![](/media/articles/custom-domains/auth0-managed.png)
+
+* **Self-Managed Certificates**: Add the TXT verification record listed in the Dashboard to your domain's DNS record.
```

Comment:
Would it possible to add an "info box" or callout here that mentions basically:
- these steps vary by domain host provider, but generally: 
- Login to your domain management service (for example GoDaddy or Google Domains).
- Create a new record and add the verification:
- For the record type, choose TXT.
- In the Name field, enter your custom domain name (for example, for login.acme.com, enter login).
- In the Time to Live (TTL) field, you can leave the default.
- In the Value field, paste the verification value from the Dashboard.
- Save the record. The changes might take a few minutes to take effect.